src/components/Card.tsx

```diff
@@ -0,0 +1,67 @@
+import React from "react";
+import {
+    Box,
+    Image,
+    Text,
+    Link,
+    Stack
+} from "@chakra-ui/react";
+
+interface Cardprops {
+    product: string
+    summary: string
+    longLine: string
+}
+
+function Card(props: Cardprops) {
+    const { product, summary, longLine } = props;
+
+    return (
+        <Box
+            p={4}
+            display={{ md: "flex" }}
+            maxWidth="32rem"
+            borderWidth={1}
+            margin={2}
+        >
+
+            <Image
+                maxWidth={'200px'}
+                margin={'auto'}
+                src="https://opencart.templatemela.com/OPC10/OPC100247/OPC1/image/catalog/Logo.png"
+                alt="Woman paying for a purchase"
```

Comment:
always think about this if you are doing a dynamic component that will consume data from props. if you found yourself doing a static data like the `src` and `alt` here, re-think again if this data also needs to be dynamic and comsumed from props?

Reply:
Same problem of source of pics and how  could call it back , know there is problem here but , can't help that